### ISSUE TITLE
text width does not follow screen width

### ISSUE THREAD

**Issue by benaryorg** [fuchsmich/notizenmd#3]
The text width is always the value of the smallest screen dimension, no matter the rotation.
In landscape mode the text just wraps at a little less than half the width.
Had a quick glance at the code but can't pin down where it comes from.

**Comment by fuchsmich** [comment]
Good idea. This might also solve another issue mentioned on openrepos.
